### ISSUE TITLE
Hyperparameter tuning (validation set)

### ISSUE THREAD

**Issue by mstow597** [markstowell15/StanceDetectionML-Brockport#3]
Implement validation set during hyperparameter tuning (finalization of model).

Review K-fold cross validation.

**Comment by mstow597** [comment]
https://machinelearningmastery.com/difference-test-validation-datasets/

Reference for difference between test set and validation set
